### PR TITLE
Low key

### PR DESCRIPTION
**Pull request by speizerj**
This merges WordpressAppPlugin into WordpressBasePlugin. Plugins have the option of being "minimal" or not, which determines the initial level of "Eagerness". Once either multiple eager plugins are installed OR multiple apps have been installed (through the main eager app), minimal plugins will convert to a regular "base" plugin. 

No matter how many eager plugins are installed, only 1 base menu will be displayed. 

The small exception to this is when the user hasn't opted in to our data collection. In that case, a menu item will be displayed for every activated plugin with the text "install {{.App.Alias}}". That menu item will take them to the page to opt-in. Once they do that for any app, those menu items will disappear and either the minimal or base menu will show.

### CODE REVIEW

**Comment by zackbloom** [comment]
🚀 🔥 

Great PR description as well!

File: includes/actions.php [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    $minimal = true;
+
+    $installs = eager_{{.Id}}_get_installs();
+    $externallyInstalled = 0;
+
+    // Don't count this current plugin when determining if more than one app is installed (for mininmal vs base purposes)
+    if ($installs) {
+      foreach ($installs as $install) {
+        if ($install['app']['id'] !== '{{.App.Id}}') {
+          $externallyInstalled++;
```
Comment (zackbloom) [inline includes/actions.php]: You can just say
```
$minimal = false;
break;
```
here, no?

File: includes/actions.php [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    // Opt-in page
+    if (!$optin) {
+      add_action('admin_menu', 'eager_{{.Id}}_activation_menu');
+    } else {
+      add_action('admin_menu', $menu);
+    }
+
+    return true;
+
+    // non admin page - embed the eager code so apps display on site
```
Comment (zackbloom) [inline includes/actions.php]: I think this comment should probably be one line lower.

File: includes/actions.php (hunk 1 above)
Comment (speizerj) [inline includes/actions.php]: yeah good call

**Pull request merged**